### ISSUE TITLE
usage of -http.pathPrefix for victoriametrics/vmalert (tested on v1.101.0)

### ISSUE THREAD

**Issue by maxospiquante** [VictoriaMetrics/VictoriaMetrics#6433]
### Is your question request related to a specific component?

vmalert

### Describe the question in detail

The documentation states, "An optional prefix to add to all the paths handled by the HTTP server." 
When I use this feature, my app is correctly exposed at /myvmalertwebapp.
Static files are correctly linked as:
`<link href="/myvmalertwebapp/vmalert/static/css/bootstrap.min.css"> <script src="/myvmalertwebapp/vmalert/static/js/jquery-3.6.0.min.js" ...`
Menu links are also correctly prefixed:
`<a class="nav-link active" href="/myvmalertwebapp/vmalert">vmalert</a>`

However, the links for flags, metrics, and reload remain hardcoded with a '/' in first char:

```
<a href="/flags">/flags</a> - command-line flags<br/>
<a href="/metrics">/metrics</a> - list of application metrics<br/>
<a href="/-/reload">/-/reload</a> - reload configuration<br/>
```

As a result, I cannot configure the vmalert webapp behind a proxy. Should I report this as a bug, or is there something I am misunderstanding?

Thx

### Troubleshooting docs

- [X] General - https://docs.victoriametrics.com/troubleshooting/
- [X] vmagent - https://docs.victoriametrics.com/vmagent/#troubleshooting
- [X] vmalert - https://docs.victoriametrics.com/vmalert/#troubleshooting

**Comment by AndrewChubatiuk** [comment]
hey @maxospiquante
running vmagent build from master branch and cannot reproduce your issue
what version are you using?

**Comment by maxospiquante** [comment]
> hey @maxospiquante running vmagent build from master branch and cannot reproduce your issue what version are you using?

Hi ! it s v1.101.0. I initially set context for k8s with values.yaml as
```
  extraArgs:
     http.pathPrefix: "/myvmalertwebapp"
```

**Comment by maxospiquante** [comment]
> @maxospiquante Added PR #6435 with a fix. Please check if solves your issue

thx a lot, lgtm

**Comment by hagen1778** [comment]
The bugfix was included into [v1.102.0-rc2](https://docs.victoriametrics.com/changelog/#v11020-rc2).
